tools/packaging/static-build/cloud-hypervisor/build-static-clh.sh

```diff
@@ -49,13 +54,32 @@ build_clh_from_source() {
     pushd "${repo_dir}"
     git fetch || true
     git checkout "${cloud_hypervisor_version}"
-    ./scripts/dev_cli.sh build --release --libc musl
+    if [ -n "${extra_build_args}" ]; then
+        info "Build cloud-hypervisor with extra args: ${extra_build_args}"
+        ./scripts/dev_cli.sh build --release --libc musl -- ${extra_build_args}
+    else
+        ./scripts/dev_cli.sh build --release --libc musl
+    fi
     rm -f cloud-hypervisor
-    cp build/cargo_target/$(uname -m)-unknown-linux-musl/release/cloud-hypervisor .
+    cp build/cargo_target/$(uname -m)-unknown-linux-musl/release/cloud-hypervisor "${binary_name}"
```

Comment:
Nit: I can see this is the minimal fix but I do wonder if it would be better to say the following (as it avoids hard-coding libc and OS names and ensures that if cargo changes "under the covers" the script won't break):

```bash
cargo install --path .
```

The message can then tell the user to find the binary at `~/.cargo/bin/${binary_name}`.

Reply:
@jodh-intel, actually, I'll end up dropping this commit from the PR as, after a conversation with Julio, I think we don't need to set a different binary name (as we may not introduce `clh-experimental`).

Your point is still valid, but I'd rather leave it for another series than touch not exactly related pieces as part of this PR.